src/components/Banner/Banner.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Banner = () => {
+	return (
```

Comment:
btw you const just do `const Banner = () => (...)` without the need to return, since there is no work done between before the render function.

Reply:
done